httpref_test.go

```diff
@@ -86,3 +101,13 @@ func TestReferences_Titles(t *testing.T) {
 	n := Statuses.Titles()
 	assert.Equal(t, 5, len(n))
 }
+
+func TestPortsConsistencyValidation(t *testing.T) {
```

Comment:
I was going to ask if this test is necessary BUT I think it actually specifies the contract for naming port mappings. Well spotted!